pyproject.toml

```diff
@@ -125,7 +131,7 @@ name = "Removals"
 showcontent = true
 
 [tool.bumpversion]
-current_version = "0.0.10"
+current_version = "0.0.9"
```

Comment:
Should we change the version to 0.0.10 (new features already added) ? current is 0.0.9